src/main/webapp/app/course/learning-goals/learning-goal-course-detail-modal/learning-goal-course-detail-modal.component.html

```diff
@@ -59,7 +59,7 @@ <h5>{{ 'artemisApp.learningGoal.learningGoalCard.connectedLectureUnits' | artemi
                             {{
                                 getLectureUnitCourseProgress(lectureUnit.id!) === undefined
                                     ? '-'
-                                    : getLectureUnitCourseProgress(lectureUnit.id!).averageScoreAchievedByStudentInLectureUnit + ' %'
+                                    : round(getLectureUnitCourseProgress(lectureUnit.id).averageScoreAchievedByStudentInLectureUnit) + ' %'
```

Comment:
Doesn't removing the `!`s in this file undo the strict template fixes?

Reply:
1ddf78e6b95cf5a3e9154c2d6b1b45661fed7dc7 Done :) good catch!